code/modules/projectiles/projectile/bullets.dm

```diff
@@ -112,6 +112,27 @@
 	..()
 	SpinAnimation()
 
+/obj/item/projectile/bullet/doorslug
+	name = "doorslug"
+	icon = 'icons/obj/meteor.dmi'
+	icon_state = "dust"
+	damage = 10
+	weaken = 2
+	stun = 2
+	hitsound = 'sound/effects/meteorimpact.ogg'
+
+/obj/item/projectile/bullet/doorslug/on_hit(atom/target, blocked = 0)
+	. = ..()
+	if(istype(target, /atom/movable))
+		var/atom/movable/M = target
+		var/atom/throw_target = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
+		var/atom/target_anchored = 0

```

Comment:
You only need ```target.anchored = 0``` on this line.  ```var/``` stuff is for defining a new variable that didn't exist before.